src/internal/types.ts

```diff
@@ -115,6 +115,7 @@ export type NetInfoSubscription = () => void;
 export interface NetInfoConfiguration {
   reachabilityUrl: string;
   reachabilityMethod?: NetInfoMethodType;
+  reachabilityHeaders?: HeadersInit;
```

Comment:
Hey @webraptor 👋 - this `HeadersInit` appears undefined, as noted in #693 - what should it be? Seem like it should be whatever valid internet headers are, perhaps `{ [index: string]: string; }` or similar? What do you think?